tests/schemas/dump_v4_default.esdl

```diff
@@ -0,0 +1,25 @@
+#
+# This source file is part of the EdgeDB open source project.
+#
+# Copyright 2023-present MagicStack Inc. and the EdgeDB authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+scalar type v3 extending ext::pgvector::vector<3>;
```

Comment:
Shouldn't we require you to have `using extension pgvector;` like with GraphQL? I think we should.

Reply:
We do; it's just configured in the test because test schemas elide the `module default` and so you aren't at the toplevel

Reply:
Got it, thanks!